docs/book/src/tasks/experimental-features/experimental-features.md

```diff
@@ -68,12 +70,28 @@ kubectl edit -n capi-system deployment.apps/capi-controller-manager
       --feature-gates=MachinePool=true,ClusterResourceSet=true
 ```
 
-Similarly, to **validate** if a particular feature is enabled, see cluster-api-provider deployment arguments by:
+Similarly, to **validate** if a particular feature is enabled, see the arguments by issuing:
 
 ```bash
 kubectl describe -n capi-system deployment.apps/capi-controller-manager
 ```
 
+Following controller manager deployments have to be edited in order to enable/disable their respective experimental features:
+
+* [MachinePools](./machine-pools.md):
+  * CAPI controller manager deployment
+  * [CABPK](https://main.cluster-api.sigs.k8s.io/reference/glossary.html?highlight=Gloss#cabpk) deployment
```

Comment:
Should we add that it could be that this also has to be enabled on the infra provider? (e.g. like it is the case for CAPD)

Reply:
This is a hard one as it may or may not be needed on the infra provider. Same with ClusterClass and I guess other flags feasibly. Maybe we should add a general note about this instead of limiting it to these two?

Reply:
I'm fine either way. I think it's fine to explicitly mention CAPD here with a note that it might be the same for other infra providers.

So basically we stick to what we do in CAPD / our reference implementation

Reply:
SGTM